### PR TITLE
add new rubocops

### PR DESCRIPTION
**Pull request by ndushay**
## Why was this change made? 🤔

She who adds rubocops saves the world entire.

## How was this change tested? 🤨

⚡ ⚠ If this change has cross service impact (e.g. changes what is written to shared file systems or how it uses APIs), ***run [integration tests](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test in [stage|qa] environment, in addition to specs. ⚡

### CODE REVIEW

File: app/lib/pre_assembly/batch.rb [hunk 1]
```diff
@@ -56,7 +56,7 @@ def run_log_msg
         project_name: project_name,
         bundle_dir: bundle_dir,
         assembly_staging_dir: Settings.assembly_staging_dir,
-        environment: ENV['RAILS_ENV']
+        environment: ENV.fetch('RAILS_ENV')
```
Comment (jcoyne) [inline app/lib/pre_assembly/batch.rb:59]: This is fine, but we probably should use `Rails.env` here. That handles the case where `RAILS_ENV` is omitted and defaults to `development`.

File: config/initializers/devise.rb [hunk 2]
```diff
@@ -6,7 +6,7 @@
   config.env_key = lambda do |env|
     if env['REMOTE_USER']
       env['REMOTE_USER']
-    elsif Rails.env.development? && ENV['REMOTE_USER']
+    elsif Rails.env.development? && ENV.fetch('REMOTE_USER')
```
Comment (jcoyne) [inline config/initializers/devise.rb:9]: Should this be `ENV.fetch('REMOTE_USER', nil)` otherwise this could raise a `KeyError` here.

**Review by jcoyne — APPROVED** [review]
Approved, pending consideration of comments.

File: config/initializers/devise.rb (hunk 2 above)
Comment (ndushay) [reply config/initializers/devise.rb:9]: ok, thanks!

File: app/lib/pre_assembly/batch.rb (hunk 1 above)
Comment (ndushay) [reply app/lib/pre_assembly/batch.rb:59]: thanks.

**Comment by ndushay** [comment]
@jcoyne - took your suggestions - thank you.

**Pull request merged**